alphagov/paas-admin
# [#154558835] Add application overview section

## What

We'd like to allow our tenants to look into basic information of their
application.

This is fairly simplistic view and may be extended over time.

## How to review

- Code review
- Run tests (should be done by travis)
- Run the application against rafal's environment
  ```
  OAUTH_AUTHORIZATION_URL="https://uaa.rafalp.dev.cloudpipeline.digital/oauth/authorize" OAUTH_TOKEN_URL="https://uaa.rafalp.dev.cloudpipeline.digital/oauth/token" OAUTH_CLIENT_ID="rafal-login" OAUTH_CLIENT_SECRET="secret" API_URL="https://api.rafalp.dev.cloudpipeline.digital" UAA_URL="https://uaa.rafalp.dev.cloudpipeline.digital/oauth/token" NOTIFY_API_KEY="not-important-for-that-story" npm start
  ```
- Navigate through orgs and spaces
- Preview Application summary
